app/uk/gov/hmrc/ctcguaranteebalancerouter/models/responses/EISResponse.scala

```diff
@@ -17,12 +17,10 @@
 package uk.gov.hmrc.ctcguaranteebalancerouter.models.responses
 
 import play.api.libs.json.Json
+import play.api.libs.json.OFormat
 
-case class EISResponse(message: String, timestamp: String, path: String) {
-  def invalidAccessCode  = message == "Not Valid Access Code for this operation"
```

Comment:
hope these are unused earlier

Reply:
Looks fine as we match on message.

Reply:
Yes, I moved this check from here just so we have one match, rather than a load of `if/else` statements.